seisbench/models/base.py

```diff
@@ -192,6 +193,113 @@ def download_metadata_callback(metadata_path):
 
         return weights
 
+    @classmethod
+    def load(cls, path):
+        """
+        Load a SeisBench model from local path.
+
+        For more information on the SeisBench model format see py:func:`save`.
+
+        :param path: Define the path to the SeisBench model directory.
+        :type path: pathlib.Path
+        :return: Model instance
+        :rtype: SeisBenchModel
+        """
+        model_files = os.listdir(path)
+
+        if sum([file.endswith("pt") for file in model_files]) > 1:
```

Comment:
There should be a check if their is actually a weights file. The json file (so far) is not mandatory.